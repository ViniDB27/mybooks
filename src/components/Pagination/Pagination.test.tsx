import render from "react-test-renderer";

import { Pagination } from "./index";

describe("<Pagination />", () => {
  const tree = render
    .create(
      <Pagination
        itensCount={1}
        itensPerPage={1}
        page={1}
        indexCurrent={1}
        onChange={() => {}}
      />
    )
    .toJSON();

  it("renders correctly", () => {
    expect(tree).toMatchSnapshot();
  });
});
