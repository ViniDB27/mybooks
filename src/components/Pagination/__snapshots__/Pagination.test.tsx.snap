// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pagination /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#0AF",
      "flexDirection": "row",
      "height": 60,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderRadius": 5,
          "height": 25,
          "justifyContent": "center",
          "width": 25,
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 22,
            "marginHorizontal": 5,
          }
        }
      >
        &lt;
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "marginHorizontal": 10,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 22,
          "marginHorizontal": 5,
          "textDecorationLine": "underline",
        }
      }
    >
      1
    </Text>
    <Text
      style={
        Object {
          "fontSize": 22,
          "marginHorizontal": 5,
        }
      }
    >
      2
    </Text>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderRadius": 5,
          "height": 25,
          "justifyContent": "center",
          "width": 25,
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 22,
            "marginHorizontal": 5,
          }
        }
      >
        &gt;
      </Text>
    </View>
  </View>
</View>
`;
